test/api/layout/execute_with_estimation_test.cpp

```diff
@@ -139,10 +139,9 @@ TEST(execute_estimation_test, many_ubs)
 ## (l*m)_tmax : Computed by l_tmax * m_tmax
 ## size : The expected total size of an tmax-HIBF
 # tmax	c_tmax	l_tmax	m_tmax	(l*m)_tmax	size
-64	1.00	1.25	1.00	1.25	90.7KiB
-128	1.22	1.24	0.69	0.85	62.2KiB
-256	1.33	1.34	0.79	1.06	71.4KiB
-# Best t_max (regarding expected query runtime): 128
+64	1.00	1.17	1.00	1.17	55.5KiB
+128	1.22	1.31	1.05	1.37	58.2KiB
+# Best t_max (regarding expected query runtime): 64
```

Comment:
size with relaxed FPR get less, which is expected. It also makes more sense to use tmax 64 for only 96 user bins.